test/CMakeLists.txt

```diff
@@ -1,10 +1,14 @@
-cmake_minimum_required(VERSION 3.14...3.22)
+#
+# This project enables the developer to build, test, install the project.
```

Comment:
Note this: The developer mode may be enabled, and any other `project_options OPT_....`